runtime-modules/council/src/lib.rs

```diff
@@ -367,6 +368,10 @@ decl_storage! { generate_storage_info
 
         /// Councilor reward per block
         pub CouncilorReward get(fn councilor_reward) config(): Balance<T>;
+
+        /// Era payou damping factor: a parameter in [0,1] that can be used to reduce the era
+        /// payout without changing the reward curve directly
+        pub EraPayoutDampingFactor get(fn era_payout_damping_factor) config(): Percent;
```

Comment:
We probably want the default/initial damping factor to be 100%

Reply:
I believe I have set it to 100% to the chainspec file

Reply:
Yes I see it. We will need to add a runtime migration to also set it to 100%.

Reply:
Prepared this https://github.com/ignazio-bovo/joystream/pull/14